stdlib/nodejs/yaml.rb

```diff
@@ -1,19 +1 @@
-# backtick_javascript: true
-
-require 'native'
-require 'nodejs/js-yaml-3-6-1'
-
-module YAML
-  @__yaml__ = `globalThis.jsyaml`
-  `var __yaml__ = #{@__yaml__}`
-
-  def self.load_path(path)
-    load(`#{File}.__fs__.readFileSync(#{path}, 'utf8')`)
-  end
-
-  def self.load(data)
-    loaded = `__yaml__.safeLoad(data)`
-    loaded = Hash.new(loaded) if native?(loaded)
-    loaded
-  end
-end
+warn "DEPRECATED: requiring nodejs/yaml is deprecated: use `require 'yaml'` instead"
```

Comment:
can you add the `require 'yaml'` just after the warning, so that `require 'nodejs/yaml'` will still work and we'll drop the whole file at the next major